packages/wallet/frontend/src/pages/transactions.tsx

```diff
@@ -323,7 +323,7 @@ const TransactionsPage: NextPageWithLayout<TransactionsPageProps> = ({
                       />
                     </Table.Cell>
                     <Table.Cell className="whitespace-nowrap">
-                      {formatDate({ date: trx.createdAt })}
+                      {formatDate({ date: trx.createdAt.toString() })}
```

Comment:
isn't it string already?
https://github.com/interledger/testnet/blob/main/packages/shared/backend/src/model.ts#L31